Makefile

```diff
@@ -52,7 +52,7 @@ ruff: # Run 'ruff' linter and print a preview of errors
 	pipenv run ruff check .
 
 safety: # Check for security vulnerabilities and verify Pipfile.lock is up-to-date
-	pipenv check
```

Comment:
Let's redo this according to https://mitlibraries.atlassian.net/browse/IN-1240

Reply:
Please see latest commit: [Replace pipenv check with pip-audit](https://github.com/MITLibraries/dspace-submission-composer/pull/173/commits/ffa6208b393661a96f66e8504d81ac7894da37d1). Thanks for the heads up!